server/src/main/resources/db/common/V92__alter_feedback_templates_add_review_flag.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE feedback_templates
+ADD COLUMN is_review boolean default false;
```

Comment:
added the field with a default of false so that current templates will remain visible